docs/DEBUG.md

```diff
@@ -1,10 +1,10 @@
 # How to debug with OpenCore
 
-For those who've hit an odd bug and unsure if it's user error or patcher, we recommend asking on the [OpenCore Patcher Paradise Discord Server](https://discord.gg/rqdPgH8xSN) for help.
+For those who've hit an odd bug and unsure if it's user error or a patcher error, we recommend asking on the [OpenCore Patcher Paradise Discord Server](https://discord.gg/rqdPgH8xSN) for help.
```

Comment:
Double user of error is a bit odd when in reference to software issue, would advise using `bug` or something similar